lib/src/define_env.dart

```diff
@@ -6,7 +6,12 @@ import 'package:dotenv/dotenv.dart' as dotEnv;
 void loadEnvFromFile(String file) {
   checkFileExists(file);
   dotEnv.load(file);
-  Platform.environment.forEach((key, value) => dotEnv.env.remove(key));
+  Platform.environment.forEach((key, value) {
+    if (dotEnv.env.remove(key) == null) return;
+    Console.setTextColor(Color.YELLOW.id);
+    Console.write(
+        "The field '$key' in the .env file was skipped because already present in 'Platform.environment' map.");
+  });
```

Comment:
This is an oversight at my end
I believe the variables in `.env` should have higher priority over `Platform.environment` map.
I Created a new issue for this here at #3

Reply:
Surely it is not said that the environment in which you generate the environment variables for dart is the same in which you are going to use them
For example CI
Ok I remove this piece of code from the PR